alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/MessageContentCollector.java

```diff
@@ -208,17 +208,17 @@ private JsonFieldAccessor createJsonAccessor(final List<JsonField<?>> notificati
                 aggregateMessageContentsForNotifications.add(foundContent);
             } else {
                 final Set<CategoryItem> categoryItems = new LinkedHashSet<>();
-                aggregateMessageContentsForNotifications.add(createAggregateMessageContent(topicItem, subTopic, categoryItems));
+                aggregateMessageContentsForNotifications.add(createAggregateMessageContent(topicItem, subTopic, categoryItems, notification));
             }
         }
         return aggregateMessageContentsForNotifications;
     }
 
-    private AggregateMessageContent createAggregateMessageContent(final LinkableItem topicItem, final Optional<LinkableItem> subTopic, final Set<CategoryItem> categoryItems) {
+    private AggregateMessageContent createAggregateMessageContent(final LinkableItem topicItem, final Optional<LinkableItem> subTopic, final Set<CategoryItem> categoryItems, AlertNotificationWrapper notification) {
```

Comment:
Shouldn't we just be passing in the creation date if that is the only field we are using from the notification?

Reply:
We used to pass just the notificationID in another method.  Then I added the date for the provider creation.  Rather than just passing each individual field I pass the actual notification in case in the future we need to grab other fields from the notification.  The dependency is on data within AlertNotificationWrapper so pass the object and allow the method to extract what it needs from the object rather than passing each individual field.  That's why I opted to use the object instead of just the date.